### ISSUE TITLE
Add More Exchanges

### ISSUE THREAD

**Issue by anighanta** [hashgraph/hedera-exchange-rate-tool#157]
**Summary**
We should also be adding more exchanges to the median calculation.

**Comment by anighanta** [comment]
List of exchanges which we don't cover that list Hbar : 

| Exchange      | Pair |
| ----------- | ----------- |
| OKEx   | Hbar/USDT        |
| LATOKEN   | Hbar/USDT        |
| Kucoin      | Hbar/USDT       |
| HitBTC   | Hbar/USD        |
| Gate.io      | Hbar/USDT       |


@lbaird Should we include all of these ?

**Comment by anighanta** [comment]
After researching a bit on the exchanges that are listed on `https://hedera.com/buying-guide`

these are a few exchanges that offer 24hr tickers for HBAR-USDT rate.

```
BitMart : https://api-cloud.bitmart.com/contract/v1/tickers?contract_symbol=HBARUSDT

Gate.io : https://api.gateio.ws/api/v4/spot/tickers?currency_pair=HBAR_USDT

Hitbit : https://api.hitbtc.com/api/3/public/ticker/HBARUSDT

BitTrue : https://openapi.bitrue.com/api/v1/ticker/24hr?symbol=HBAR3SUSDT

Huobi : https://api.huobi.pro/market/detail?symbol=hbarusdt
```

Will start with these and integrate them into ERT

**Comment by lbaird** [comment]
Great. Let’s add those, now.

In the future, we should also add exchanges that don’t have that pair, but
do have both HBAR-BTC and BTC-USDT.

**Comment by anighanta** [comment]
Yup, created new issue https://github.com/hashgraph/hedera-exchange-rate-tool/issues/167 for tracking that feature.

**Comment by anighanta** [comment]
Huobi's api only returns the latest asks and bibs for HBAR-USD. 

`https://api.huobi.pro/market/detail/merged?symbol=hbarusdt` returns 
```
{"ch":"market.hbarusdt.detail.merged","status":"ok","ts":1639988550923,"tick":{"id":203157525848,"version":203157525848,"open":0.303986,"close":0.319126,"low":0.295542,"high":0.334499,"amount":1.016234755155054E7,"vol":3227770.2944399095,"count":28322,"bid":[0.318934,39.15],"ask":[0.319217,1700.0]}}
```

None of the Huobi's api return the price of HBAR. Instead we can get 
1. asks and bids and find a mean 
2. or the latest trades which will have the price of the HBAR that they traded USD for. 

Both of them could be manipulated I think.

**Comment by lbaird** [comment]
The latest trade is the price of HBAR. That’s the definition of “price” for
any market. It isn’t the average of asks and bids. It’s the price of the
most recent trade.
